detekt/dependencies.bzl

```diff
@@ -13,6 +13,31 @@ def rules_detekt_dependencies():
     Users should call this macro in their `WORKSPACE` file.
     """
 
+    # Pkg
+
+    rules_pkg_version = "0.2.4"
+    rules_pkg_sha = "4ba8f4ab0ff85f2484287ab06c0d871dcb31cc54d439457d28fd4ae14b18450a"
+
+    maybe(
+        http_archive,
+        name = "rules_pkg",
+        url = "https://github.com/bazelbuild/rules_pkg/releases/download/{v}/rules_pkg-{v}.tar.gz".format(v = rules_pkg_version),
+        sha256 = rules_pkg_sha,
+    )
+
+    # Stardoc
+
+    rules_java_version = "0.4.0"
+    rules_java_sha = "36b8d6c2260068b9ff82faea2f7add164bf3436eac9ba3ec14809f335346d66a"
+
+    maybe(
+        repo_rule = http_archive,
+        name = "io_bazel_stardoc",
+        sha256 = rules_java_sha,
+        strip_prefix = "stardoc-{}".format(rules_java_version),
+        url = "https://github.com/bazelbuild/stardoc/archive/{}.zip".format(rules_java_version),
+    )
```

Comment:
Is there a reason to have package and Stardoc rules as explicit dependencies? Shouldn’t Kotlin rules provide them as transitive ones?

Reply:
You would hope but it seems like Bazel wants you to be explicit about transitive dependencies in WORKSPACE files. If you see an obvious way to work around this I can make the changes, but I wasn't able to get `rules_detekt` building against `rules_kotlin` alpha without being explicit about the transitives.

https://docs.bazel.build/versions/master/external.html#transitive-dependencies

Reply:
I think the root of the issue here is we're not using the prebuilt version. Switching to https://github.com/bazelbuild/rules_kotlin/releases/download/v1.5.0-alpha-3/rules_kotlin_release.tgz (`eeae65f973b70896e474c57aa7681e444d7a5446d9ec0a59bb88c59fc263ff62`) would 1) stop unnecessarily building the Kotlin builder 2) remove the need for these dependencies.

Reply:
That cleared that up. Thanks @Kernald